agent/fw_drupal.c

```diff
@@ -476,18 +490,16 @@ static void nr_drupal_wrap_hook_within_module_invoke_all(
     return;
   }
 
-  rv = module_invoke_all_parse_module_and_hook(
-      &module, &module_len, hook_name,
-      hook_len, func);
+  rv = module_invoke_all_parse_module_and_hook(&module, &module_len, hook_name,
+                                               hook_len, func);
 
   if (NR_SUCCESS != rv) {
     return;
   }
 
-  nr_php_wrap_user_function_drupal(
-      nr_php_function_name(func), nr_php_function_name_length(func), module,
-      module_len, hook_name,
-      hook_len TSRMLS_CC);
+  nr_php_wrap_user_function_drupal(nr_php_function_name(func),
+                                   nr_php_function_name_length(func), module,
+                                   module_len, hook_name, hook_len TSRMLS_CC);
```

Comment:
This is just the formatting change, right?

Reply:
yes